pkg/domain/infra/abi/containers.go

```diff
@@ -77,10 +77,23 @@ func getContainersAndInputByContext(all, latest bool, names []string, filters ma
 			}
 		}
 	case latest:
-		ctr, err = runtime.GetLatestContainer()
-		if err == nil {
-			rawInput = append(rawInput, ctr.ID())
-			ctrs = append(ctrs, ctr)
+		if len(names) > 0 {
```

Comment:
Not sure if this is the best way to determine whether we want the latest logs from a pod or a container, but it seems to work.